style/modules/artworkThumbnail.css

```diff
@@ -0,0 +1,27 @@
+.artworkThumbnail {
+    display: block;
+    overflow: hidden;
+    transition: opacity 0.5s ease;
```

Comment:
@kingtrentiii is this for placing the map? if so, what is the transition line meant to do? and the hover opacity, and overflow none...

If I understand the point of this module, then it looks like there are things from `muralThumbnail` in here that can come out. you should go through each line in this file, comment it out, and if there's no change it can probably be deleted.